lib/org/chromium/webidl/ast/Argument.js

```diff
@@ -24,6 +24,11 @@ foam.CLASS({
       class: 'Boolean',
       name: 'isVariadic',
     },
+    {
+      class: 'String',
+      name: 'id',
+      factory: function() { return this.getName(); },
```

Comment:
This seems like a good rationale. Let's make sure it appears in the documentation string.